Content.Server/SimpleStation14/Silicon/Systems/SiliconMiscSystem.cs

```diff
@@ -1,20 +1,43 @@
 using Content.Shared.SimpleStation14.Silicon.Components;
 using Content.Shared.Bed.Sleep;
+using static Content.Shared.Repairable.SharedRepairableSystem;
+using Content.Server.Body.Components;
+using Content.Server.Body.Systems;
 
-namespace Content.Server.SimpleStation14.Silicon.Misc;
+namespace Content.Server.SimpleStation14.Silicon.Sytstems;
```

Comment:
This would require a more major refactoring of the Silicon files and name spaces. I'll do it at some point, but it probably shouldn't hold up this pr.

Reply:
what